src/qt/bitcoin.cpp

```diff
@@ -676,7 +682,7 @@ int main(int argc, char *argv[])
     app.createOptionsModel(IsArgSet("-resetguisettings"));
 
     // Subscribe to global signals from core
-    uiInterface.InitMessage.connect(InitMessage);
+    FIXME_IMPLEMENT_IPC_VALUE(uiInterface).InitMessage.connect(InitMessage);
```

Comment:
How would you handle `uiInterface` in this model, e.g. signals from the server to the client?